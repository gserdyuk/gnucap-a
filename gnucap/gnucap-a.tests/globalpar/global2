#!/bin/bash
GNUCAP=../src/gnucap

#export EF_PROTECT_BELOW=1; 
export EF_PROTECT_FREE=1; $GNUCAP -b global2.gckt  | tee global2.out
