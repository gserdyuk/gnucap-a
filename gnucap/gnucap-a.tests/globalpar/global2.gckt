* global parameter test
.options parhier=local
.param a='3'

.subckt MODULE1 m1p1 m1p2

.subckt MODULE2 p1 p2
.param b='4'
v1 p1 0 dc 'a'
v2 p2 0 dc 'b'
.ends

xlower m1p1 m1p2 MODULE2

.ends

xmain nd1 nd2 MODULE1


*.list
.print op v(nd*)
*.print  all
.op
.end

