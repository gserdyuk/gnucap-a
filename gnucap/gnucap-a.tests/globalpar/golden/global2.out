Gnucap gnucap-arails  2009.12.07 RCS 26.136
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
* global parameter test 
#           v(nd1)     v(nd2)    
 27.        3.         4.        
